module/go.mod

```diff
@@ -20,47 +22,46 @@ require (
 	github.com/spf13/cast v1.5.0
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/viper v1.15.0
-	github.com/stretchr/testify v1.8.1
-	github.com/tendermint/tendermint v0.37.0-rc2
-	github.com/tendermint/tm-db v0.6.7
-	google.golang.org/genproto v0.0.0-20221227171554-f9683d7f8bef
-	google.golang.org/grpc v1.52.0
+	github.com/stretchr/testify v1.8.2
+	github.com/tendermint/tendermint v0.35.9
```

Comment:
Needed for the v1 types files. 

We may just removed them since we dont have to migrate from v1 to v2